cli/src/commands/extensions/api.rs

```diff
@@ -361,8 +361,12 @@ async fn parse_lockfile(
     // Attempt to parse as requested lockfile type.
     let parsed = parse::parse_lockfile(lockfile, lockfile_type.as_deref())?;
 
+    // PackageLock should report LockfileFormat and not PackageType. This is just a
+    // placeholder until that can be updated.
+    let package_type = parsed.packages.first().map(|p| p.package_type).unwrap_or(PackageType::Npm);
+
     Ok(PackageLock {
-        package_type: parsed.package_type,
+        package_type,
         packages: parsed.packages.into_iter().map(PackageSpecifier::from).collect(),
     })
```

Comment:
I'll create an issue to track this transition

Reply:
What prevents us from fixing this immediately?

Reply:
It would be a breaking change for extensions.

After looking through the extensions, the only official or community extension that this would break is [`sbom`](https://github.com/phylum-dev/community-extensions/blob/da1e4b324c76b11f1ce9846574c75b34cc1e9a47/sbom/main.ts#L127). I'm surprised it's only one. But I'd still rather separate that change since it is breaking.

Reply:
> How would you suggest migration?

That's the question I intend to answer in the separate issue... I don't have a clear answer right now and I didn't want to hold up this PR until I've figured that out.